tutorials/mission_specific_analyses/nicer/data-analysis-nicer.md

```diff
@@ -0,0 +1,598 @@
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.17.3
+kernelspec:
+  name: heasoft
+  display_name: heasoft
+  language: python
+title: Getting started with NICER data
+---
+
+# Getting started with NICER data
+
+## Learning Goals
+
+By the end of this tutorial, you will be able to:
+- Download NICER observation data files for a given ObsID.
+- Process and clean raw NICER data into "level 2" (science ready) products.
+- Extract a spectrum, fit a model to it using pyXspec, and plot the results.
+- Extract a light curve, account for good-time-intervals, and plot it.
+
+## Introduction
+
+In this tutorial, we will go through the steps of analyzing a NICER observation (with ObsID 4020180445) of
+'PSR B0833-45' using `heasoftpy`.
+
+### Inputs
+
+- The NICER ObsID, 4142010107, of the data we will process (an observation of a pulsar, **PSR B0833-45**).
+
+### Outputs
+
+- Processed NICER "level 2" data files.
+- A source and background spectrum.
+- A source light curve.
+- Fitted spectral model parameters.
+- Visualizations of the spectrum and light curve.
+
+### Runtime
+
+As of 21st October 2025, this notebook takes ~6m to run to completion on Fornax using the 'Default Astrophysics' image and the 'small' server with 8GB RAM/ 2 cores.
+
+## Imports
+
+```{code-cell} python
+import contextlib
+import os
+import shutil
+
+import heasoftpy as hsp
+import matplotlib.pyplot as plt
+import numpy as np
+import xspec as xs
+from astropy.io import fits
+from astropy.table import Table
+from astroquery.heasarc import Heasarc
+from heasoftpy.nicer import nicerl2, nicerl3_lc, nicerl3_spect
+from matplotlib.ticker import FuncFormatter
+```
+
+## Global Setup
+
+### Functions
+
+```{code-cell} python
+:tags: [hide-input]
+
+# This cell will be automatically collapsed when the notebook is rendered, which helps
+#  to hide large and distracting functions while keeping the notebook self-contained
+#  and leaving them easily accessible to the user
+```
+
+### Constants
+
+```{code-cell} python
+:tags: [hide-input]
+
+# NICER ObsID that we will use for this example.
+OBS_ID = "4020180445"
+SRC_NAME = "PSR-B0833-45"
+
+# The name of the HEASARC table that logs all NICER observations
+HEASARC_TABLE_NAME = "nicermastr"
+```
+
+### Configuration
+
+```{code-cell} python
+:tags: [hide-input]
+
+# Set up the path of the directory into which we will download NICER data
+if os.path.exists("../../../_data"):
+    ROOT_DATA_DIR = os.path.join(os.path.abspath("../../../_data"), "NICER", "")
+else:
+    ROOT_DATA_DIR = "NICER/"
+
+# Get the absolute path to the download directory
+ROOT_DATA_DIR = os.path.abspath(ROOT_DATA_DIR)
+
+# Make sure the download directory exists.
+os.makedirs(ROOT_DATA_DIR, exist_ok=True)
+
+# Setup path and directory into which we save output files from this example.
+OUT_PATH = os.path.abspath("NICER_output")
+os.makedirs(OUT_PATH, exist_ok=True)
+
+# -------- Get geomagnetic data ---------
```

Comment:
It may make sense to put this in the section where it is needed. Adding it here may confuse a newbie who does not know what this is about.